templates/web/base/admin/update_edit.html

```diff
@@ -55,10 +55,42 @@
 </li>
 <li>[% loc('Cobrand:') %] [% update.cobrand %]</li>
 <li>[% loc('Cobrand data:') %] [% update.cobrand_data OR loc('None') %]</li>
+[% IF update.external_id~%]
+  <li>[% loc('External ID:') %] [% update.external_id %] </li>
+[%~END %]
 <li>[% loc('Created:') %] [% PROCESS format_time time=update.created %]</li>
+<li>[% loc('Confirmed: ') ~%]
+  [%~ IF update.confirmed ~%]
+    [% PROCESS format_time time=update.confirmed %]</li>
+  [% ELSE ~%]
+    Not confirmed
+  [% END %]
 [% IF update.get_extra_metadata('external_status_code') %]
 <li>[% loc('External status code:') %] [% update.get_extra_metadata('external_status_code') | html %]</li>
 [% END %]
+<li>[% loc('Send state:') %] [% update.send_state %]</li>
+[% IF update.send_fail_count > 0 AND update.send_state != 'sent' AND update.send_state != 'skipped' %]
```

Comment:
Note on reports, this send fail information is only shown to superusers, so presumably the same should apply here.